client/src/components/ItemPanel.vue

```diff
@@ -9,14 +9,13 @@
 
       <v-spacer />
 
-      <v-tooltip top>
+      <v-tooltip top v-if="anySelected">
         <template v-slot:activator="{ on }">
           <v-scroll-x-transition>
```

Comment:
Does it work if the transition is above the tooltip?

Reply:
Unfortunately, no :crying_cat_face:

Any other ideas?